.github/workflows/main.yml

```diff
@@ -1,31 +1,22 @@
-name: "Main"
+name: Build (experimental package)
+
 on:
   pull_request:
   push:
-    paths-ignore:
-      - "README.md"
+    branches:
+      - develop
 
 jobs:
-  build-test:
-    runs-on: ubuntu-latest
-    name: Build test
-    steps:
-      - uses: actions/checkout@v2
-      - run: |
-          npm install
-          npx @dappnode/dappnodesdk build --skip_save
-
-  release:
-    name: Release
+  build:
     runs-on: ubuntu-latest
-    needs: build-test
     steps:
-      - uses: actions/checkout@v2
-      - name: Publish
+      - uses: actions/checkout@v4
+      - name: Build and upload to IPFS
         run: |
           npm install
-          npx @dappnode/dappnodesdk publish patch --dappnode_team_preset
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          DEVELOPER_ADDRESS: "0x7305356ad936A06c4ea5DF45AD5E5C3ff9Db818E"
+          npx @dappnode/dappnodesdk build \
+            --provider "remote" \
+            --verbose | tee build.log
 
+          echo "DNP (DAppNode Package) built and uploaded" >> $GITHUB_STEP_SUMMARY
```

Comment:
It would be nice if you could improve the pipeline so it filters in the build.log the ipfs hash and then notifies the IPFS hash through Zulip or store it in a github environment variable
```
gh variable set LATEST_IPFS ${ipfs_hash}
```

And create a second workflow which would read that variable `${{ vars.NEXT_RELEASE_TYPE }}` and would publish the ipfs hash in the dappnode store.